.github/workflows/stacks-blockchain.yml

```diff
@@ -35,8 +35,18 @@ jobs:
           status: STARTING
           color: warning
 
-  # Run tests
-  test:
+  # Run large genesis test
+  large-genesis:
```

Comment:
💅  what's our take on the semantic `full-genesis` vs `sampled-genesis`/`light-genesis`?

Reply:
That works for me -- I'll update the names in the workflow.